src/vue/pages/VerificationCorporate.vue

```diff
@@ -261,13 +257,18 @@ export default {
           this.kvEntryCorporateRoleId
         )
         await Api.api.postOperations(operation)
+        if (this.kycState === REQUEST_STATES_STR.pending) {
+          Bus.success('verification-form.request-updated-msg')
+        }
         while (this.kycState !== REQUEST_STATES_STR.pending) {
           await this.loadKyc()
         }
       } catch (e) {
-        this.enableForm()
         ErrorHandler.process(e)
       }
+      this.isFormSubmitting = false
+      this.hideConfirmation()
```

Comment:
mm, breaks SRP. I would recommend restoring of `@ok="hideConfirmation() || submit()"` at line 123 but modifying it like
```
v-if="formMixin.isConfirmationShown || isFormSubmitting"
`@ok="submit() || hideConfirmation()"` (change order)
```
is it feasible?

Reply:
I think that such code-style is excess. By click on the button, you must do only `single` action, but this action may trigger other actions. Ya, your example will work fine, but do we need it? I think no.

Reply:
in the current way we smeared our form confirmation showing/hiding logic across the component, not so cool either. Got an another idea, we can extract them to a method like (namings is up to you):
```
submitAndCloseConfirmation() {
  submit()
  closeConfirmation()
}
```
it still breaks SRP but a bit more explicit

Reply:
But does we really need it? `showConformation` and `hideConfirmation` are single methods and we can use just like common methods. I cannot see any reason to worry about `form confirmation showing/hiding logic across the component` because these are common methods that we can use everywhere across the component without any worries.

Reply:
Because it breaks single responsibility principle